assignments/array-methods.js

```diff
@@ -55,22 +55,52 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
+
+
+
 let fullName = [];
+
+
+
+
+runners.forEach(runner => fullName.push(`${runner.first_name} ${runner.last_name}`));
+
 console.log(fullName);
 
+
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
+
 let allCaps = [];
-console.log(allCaps); 
 
+const allCap = runners.map( function(currentValue){
+    return currentValue.first-name.toUpperCase();
+  
+});
+  
+console.log(allCaps);
 // ==== Challenge 3: Use .filter() ====
 // The large shirts won't be available for the event due to an ordering issue.  Get a list of runners with large sized shirts so they can choose a different size. Return an array named largeShirts that contains information about the runners that have a shirt size of L and log the result
 let largeShirts = [];
-console.log(largeShirts);
+const shirts = runner.filter((currentValue) =>{
+    return currentValue.shirt_size === "L";
+  });
+  
+  console.log(largeShirts);
```

Comment:
be watchful of what you're defining and assigning here: you already defined `largeShirts` to be an empty array, remember that you don't have to declare it again with const or let when running filter, and you also call it just `shirts` on that line